stream-chat-android-client/src/main/java/io/getstream/chat/android/client/channel/ChannelMessagesUpdateLogic.kt

```diff
@@ -23,9 +23,13 @@ import io.getstream.chat.android.models.Message
 @InternalStreamChatApi
 public interface ChannelMessagesUpdateLogic {
```

Comment:
Can we make this interface `internal`? 🤔

Reply:
We can't, it is used between modules =|.